.mailmap

```diff
@@ -9,6 +9,7 @@
 Aaron Bassett <abassett@gmail.com>
 Abhishek Lekshmanan <abhishek.lekshmanan@ril.com> <abhishek.lekshmanan@gmail.com>
```

Comment:
Ah ok, what is to be done here then, I'm ok with closing this. not that much of a deal, anyway most of the commits dont use this email id, only the `reviewed-by` since that is added by the dev. doing the merge. (for eg. in 10.1.2 contributor credits it appeared in reviewed-by mostly)